libs/deal-ticket/src/components/deal-ticket-validation/validate-margin.tsx

```diff
@@ -43,11 +41,7 @@ export const ValidateMargin = ({
           )}`}
         </p>
       </div>
-      <DepositDialog
-        depositDialog={depositDialog}
-        setDepositDialog={setDepositDialog}
-        assetId={id}
-      />
+      <DepositDialog />
```

Comment:
chore (blocking): now that we have some sharable state and the dialogs are rendered in the dialogs-container we should be able to delete this

Reply:
Removed